plugins/search-backend-module-elasticsearch/src/engines/ElasticSearchSearchEngineIndexer.ts

```diff
@@ -135,7 +138,12 @@ export class ElasticSearchSearchEngineIndexer extends BatchSearchEngineIndexer {
           {
             remove: { index: this.constructIndexName('*'), alias: this.alias },
           },
-          { add: { index: this.indexName, alias: this.alias } },
+          {
+            add: {
+              index: this.indexName,
+              aliases: [this.alias, this.removableAlias],
```

Comment:
Good point! I updated the change to only apply the "removable" alias to indices that are in fact stale. Under normal operations, you should never see this alias since indices with this alias should be removed right after. If deletion fails then the dangling indices will persist with this alias and anyone inspecting from the outside should be able to assume these are in fact removable.

Reply:
Niiiice.  This is awesome.